tests/core/core-dump/task.yaml

```diff
@@ -7,16 +7,17 @@ details: |
 execute: |
   # To get VERSION_ID defined
   . /etc/os-release
+  core_dump_d=$TESTSLIB/snaps/core-dump-snap
   if [ "$VERSION_ID" = 16 ]; then
-      sed -i '/base: ##BASE##/d' core-dump-snap/meta/snap.yaml
+      sed -i '/base: ##BASE##/d' "$core_dump_d"/meta/snap.yaml
   else
       BASE=core"$VERSION_ID"
-      sed -i "s/##BASE##/$BASE/" core-dump-snap/meta/snap.yaml
+      sed -i "s/##BASE##/$BASE/" "$core_dump_d"/meta/snap.yaml
```

Comment:
is has to be reverted during restore

Reply:
Good point, what I've done now is to copy inside the test folder before modifying.